src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -599,6 +639,25 @@ private BoundExpression CreateAndPopulateArray(BoundCollectionExpression node, A
                             _factory.Binary(BinaryOperatorKind.Addition, indexTemp.Type, indexTemp, _factory.Literal(1)),
                             isRef: false,
                             indexTemp.Type));
+                },
+                (sideEffects, targetArray, spreadElement, rewrittenSpreadOperand) =>
```

Comment:
>targetArray [](http://example.com/codeflow?start=30&length=11)

Consider calling this `arrayTemp`, for consistency with lambda expression above, and to shadow the `arrayTemp` from outside this lambda expression.